moped-editor/src/views/projects/newProjectView/DefineProjectForm.js

```diff
@@ -89,8 +89,8 @@ const DefineProjectForm = ({
             type="text"
             fullWidth
             value={projectDetails.project_description}
-            error={descriptionError}
-            helperText="Required"
+            error={descriptionError !== null}
+            helperText={descriptionError?.message || "Required"}
```

Comment:
i dont see any reason to not have this fallback but just to be sure, if the description is length 0 then `descriptionError.message` would already be "Required" from the logic set in `NewProjectView`?

Reply:
good catch! i'll update. thank you, Rose!!! 🙏